src/vendorcode/mediatek/mt8192/include/dramc_ch0_reg.h

```diff
@@ -3388,10 +3388,10 @@
 	#define SHU4_PIPE_READ_START_EXTEND2		BIT(29)
 	#define SHU4_PIPE_DLE_LAST_EXTEND1		BIT(30)
 	#define SHU4_PIPE_READ_START_EXTEND1		BIT(31)
-#define SHU4_TEST1					0x00001a7c
-	#define SHU4_TEST1_LATNORMPOP			GENMASK(12, 8)
-	#define SHU4_TEST1_DQSICALBLCOK_CNT		GENMASK(22, 20)
-	#define SHU4_TEST1_DQSICALI_NEW			BIT(23)
+#define SHu4EST1					0x00001a7c
```

Comment:
@IgorBagnucki and these

Reply:
No, these changes were unintentional. Removed them